src/lib.rs

```diff
@@ -121,12 +121,14 @@
 //!
 //! # Features
 //!
-//! fast-xml supports 2 additional features, non activated by default:
-//! - `encoding`: support non utf8 XMLs
-//! - `serialize`: support serde `Serialize`/`Deserialize`
```

Comment:
Slightly concerned about the additional supply chain risk for a seemingly small benefit - all users now have to compile `document-features` just to use the library.  Maybe it works as a dev-dependency?  That would be a bit better.

I'm not strongly against, but personally I think I would just update these docs.

Reply:
Unfortunately, it does not work as dev-dependency. Fortunately, it is [very small library](https://github.com/slint-ui/document-features/blob/master/lib.rs) that is used only when documentation is generated, so I think we can live with that. If in the future it will block us to move on, it will be removed